frontend/javascript/components/docupload/file-review.vue

```diff
@@ -119,10 +119,9 @@ export default {
   },
   mounted() {
     if (this.hasSubMenu) {
-      // eslint-disable-next-line no-new
-      new Dropdown(
-        document.getElementById(`docupload-dropdown-${this.attachment.id}`)
-      )
+      window.setTimeout(() => {
```

Comment:
why the timeout? for me, it works without it

Reply:
Indeed!